### ISSUE TITLE
a11y issues on Component pages

### ISSUE THREAD

**Issue by mahoneycm** [uswds/uswds-site#1651]
Related to #1109 

## Pages to fix:

- [ ] /components/
- [ ] /components/icons

### Icons Page

```js
    id: 'aria-allowed-attr',
    impact: 'serious',
    tags: [ 'cat.aria', 'wcag2a', 'wcag412' ],
    description: "Ensures ARIA attributes are allowed for an element's role",
    help: 'Elements must only use allowed ARIA attributes',
    helpUrl: 'https://dequeuniversity.com/rules/axe/4.4/aria-allowed-attr?application=axeAPI',
```

![image](https://user-images.githubusercontent.com/25211650/173403324-6230d7c4-c9f1-4b71-8062-5a1b6dd97454.png)

#### Solution: 

> Fix the following:
> aria-label attribute cannot be used on a span with no valid role attribute.

----

### Component Page

_This also appears as one failure on components/icons_

```js
    id: 'aria-allowed-role',
    impact: 'minor',
    tags: [ 'cat.aria', 'best-practice' ],
    description: 'Ensures role attribute has an appropriate value for the element',
    help: 'ARIA role should be appropriate for the element',
    helpUrl: 'https://dequeuniversity.com/rules/axe/4.4/aria-allowed-role?application=axeAPI',
```

![image](https://user-images.githubusercontent.com/25211650/173403442-9a54a471-867d-45e3-930c-7198ae44733c.png)

#### Solution

>Fix the following:
aria-label attribute is not well supported on a div with no valid role attribute.

## Steps to reproduce the issue

1. Run `npm run axe`
2. Script stops at first failure
3. Remediate failure or remove failing path from `const PAGES` array on `run-axe.js` to view second failure
